spec/index.html

```diff
@@ -2359,9 +2359,7 @@ <h3>JsonLdProcessor</h3>
                 <a data-cite="JSON-LD11-API#rdf-to-object-conversion">RDF to Object Conversion</a> algorithm
                 before step 2.6 as follows:
                 <blockquote>
-                  Otherwise, if both the {{JsonLdOptions/useNativeTypes}}
-                  and {{JsonLdOptions/extendedYAML}} flags are set
-                  and the
+                  Otherwise, if both the {{JsonLdOptions/useNativeTypes}} flag is set and {{JsonLdOptions/processingMode}} parameter is `yaml-ld-extended` and the
                   <a>datatype IRI</a>
```

Comment:
```suggestion
                  Otherwise, if the {{JsonLdOptions/useNativeTypes}} flag is set,
                  the {{JsonLdOptions/processingMode}} parameter is `yaml-ld-extended`,
                  and the <a>datatype IRI</a>
```